flexget/plugins/sites/torrentday.py

```diff
@@ -0,0 +1,184 @@
+from future.moves.urllib.parse import quote
+
+import re
+import logging
+
+from requests.exceptions import RequestException
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.plugin import PluginError
+from flexget.plugins.internal.urlrewriting import UrlRewritingError
+from flexget.utils import requests
+from flexget.utils.soup import get_soup
+from flexget.utils.search import torrent_availability, normalize_unicode
+from flexget.utils.tools import parse_filesize
+
+log = logging.getLogger('torrentday')
+
+CATEGORIES = {
+    'all': 0,
+    # Movies
+    'mov480p': 25,
+    'movHD': 11,
+    'movBD': 5,
+    'movDVD': 3,
+    'movMP4': 21,
+    'movNonEnglish': 22,
+    'movPACKS': 13,
+    'movSDx264': 44,
+    'movX265': 48,
+    'movXVID': 1,
+
+    # TV
+    'tv480p': 24,
+    'tvBRD': 32,
+    'tvDVD': 31,
+    'tvDVDrip': 33,
+    'tvMOBILE': 46,
+    'tvPACKS': 14,
+    'tvSDx264': 26,
+    'tvHDx264': 7,
+    'tvX265': 34,
+    'tvXVID': 2
+}
+
+
+class UrlRewriteTorrentday(object):
+    """
+        Torrentday urlrewriter and search plugin.
+
+        torrentday:
+          uid: xxxxxxxxxxxxx  (required)    NOT YOUR LOGIN. find this in your browser's cookies
+          passkey: xxxxxxxxx  (required)    NOT YOUR PASSWORD. see previous
+          cfduid: xxxxxxxxxx  (required)    AGAIN IN THE COOKIES
+          rss_key: xxxxxxxxx  (required)    get this from your profile page
+          category: xxxxxxxx
+
+          Category can be one of 
+            ID from browsing site OR 'name'
+            movies:
+              mov480p, movHD, movBD, movDVD,
+              movMP4, movNonEnglish, movPACKS,
+              movSDx264, movX265, movXVID
+            tv:
+              tv480p, tvBRD, tvDVD, tvDVDrip,
+              tvMOBILE, tvPACKS, tvSDx264, 
+              tvHDx264, tvX265, tvXVID
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'rss_key': {'type': 'string'},
+            'uid': {'type': 'string'},
+            'passkey': {'type': 'string'},
+            'cfduid': {'type': 'string'},
+            'category': {
+                'oneOf': [
+                    {'type': 'integer'},
+                    {'type': 'string', 'enum': list(CATEGORIES)},
+                ]
+            },
+        },
+        'required': ['rss_key', 'uid', 'passkey', 'cfduid'],
+        'additionalProperties': False
+    }
+
+    # urlrewriter API
+    def url_rewritable(self, task, entry):
+        url = entry['url']
+        if url.find('.torrent'):
+            return False
+        if url.startswith('https://www.torrentday.com'):
+            return True
+        return False
+
+    # urlrewriter API
+    def url_rewrite(self, task, entry):
+        if 'url' not in entry:
+            log.error("Didn't actually get a URL...")
+        else:
+            log.debug("Got the URL: %s" % entry['url'])
```

Comment:
You can pass the args to the logger and let it do the string formatting ie. comma instead of %